### PR TITLE
forceios script updates for Cordova 3.9.2

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

File: node/forceios.js [hunk 1]
```diff
@@ -118,8 +118,8 @@ function createHybridApp(config) {
 
     shelljs.exec('cordova create "' + projectDir + '" ' + config.companyid + ' ' + config.appname);
     shelljs.pushd(projectDir);
-    shelljs.exec('cordova platform add ios');
-    shelljs.exec('cordova plugin add https://github.com/forcedotcom/SalesforceMobileSDK-CordovaPlugin');
+    shelljs.exec('cordova platform add ios@' + cordovaPlatformVersion);
+    shelljs.exec('cordova plugin add https://github.com/forcedotcom/SalesforceMobileSDK-CordovaPlugin#unstable');
```
Comment (kchitalia) [inline node/forceios.js]: Should we add a line to install the minimum supported CLI version by default?

Comment (bhariharan) [inline node/forceios.js]: Developers might have different versions of CLI installed. Installing CLI across different platforms is too painful to support. Besides, I wouldn't go that far to step on a developer's toes.

**Comment by wmathurin** [comment]
:+1:

File: node/forceios.js (hunk 1 above)
Comment (kchitalia) [inline node/forceios.js]: You are right. No point in forcing developers to use a particular CLI version.

**Pull request merged**